Automattic/wp-calypso
# Remove unused composite-checkout-wpcom package

#### Changes proposed in this Pull Request

This removes the `composite-checkout-wpcom` package which should have been removed in #40596

#### Testing instructions

This should have no effects. Run calypso and visit composite checkout and make sure nothing is obviously broken.
